### DISCUSSION TITLE
AMG8833 TO ESP32s

### DISCUSSION THREAD

**Discussion by LunaVulpine8309** [TheRealWaldo/esp8266-amg8833#12] [Q&A]
Hello, so i have an issue here. I do not know how to code AMG8833 (thermal camera) to ESP32 (node mcu) can anyone help me? thank you. I did found a video about in youtube but the person does not give any explaination of how to the coding.

**Comment by TheRealWaldo** [comment]
You may want to see https://github.com/TheRealWaldo/AMG8833-ESPHOME/discussions/10 for some inspiration.

Another resource would be the documentation for platformio: https://docs.platformio.org/en/latest/

**Reply by LunaVulpine8309** [reply]
Thank you so much sir! I appreciate your help :)
